pytest_splunk_addon/tools/cim_field_report.py

```diff
@@ -0,0 +1,423 @@
+import os.path
+import sys
+import logging
+import json
+import argparse
+import re
+import glob
+import time
+import traceback
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
```

Comment:
Doc tests are not failing on the latest commit on main, so I assume the reason is in this PR.